R/srcrefs.R

```diff
@@ -110,6 +110,9 @@ srcrefs.R6ClassGenerator <- function(x, ..., srcref_names = NULL) {
 #' @importFrom utils getSrcref
 #' @rdname srcrefs
 srcrefs.MethodDefinition <- function(x, ..., srcref_names = NULL) {
+  # catch methods in methods tables from packages without srcref data
+  if (is.null(sr <- getSrcref(x))) return(x)
```

Comment:
I'm not necessarily sure what's the purpose of that assignment here. I don't think we are using it anywhere

Reply:
Ah good catch - I should have returned `sr`, not `x` :facepalm: